SharePointFramework/PortfolioWebParts/src/components/ProjectList/useProjectList.ts

```diff
@@ -81,7 +81,15 @@ export const useProjectList = (props: IProjectListProps) => {
       .filter((project) => state.selectedVertical.filter(project, state))
       .filter((project) =>
         _.any(Object.keys(project), (key) => {
-          const value = project[key]
+          let value
```

Comment:
Kan vi ikke bare søke i en stringified versjon av `value`?

cc @Remi749 @tarjeieo

Reply:
Works for me

Reply:
Godt poeng. Mulig det blir match i object keys f.eks.

ChatGPT sier følgende:

<img width="998" alt="image" src="https://github.com/Puzzlepart/prosjektportalen365/assets/7606007/0974dc2d-c26f-448f-b5b1-d43d4f00a37d">

Det ser ikke dumt ut.

Reply:
Sikkert ikke dumt, men blir det ikke akkurat det samme jeg gjør? Bare en annen måte å samle items på...